src/app/components/dynamic-table/dynamic-table.component.ts

```diff
@@ -25,23 +25,28 @@ export class DynamicTableComponent
     @Input() public hasTableHeader: boolean;
     @Input() public tableSource?: TableSourceInterface;
     @Input() public idTable = "";
-    @Output() public onSelectRowEmit: EventEmitter<string> = new EventEmitter();
+    @Output() public selectRowEmit: EventEmitter<string> = new EventEmitter();
 
-    /* eslint-disable  @typescript-eslint/no-explicit-any */
-    public dataSource = new MatTableDataSource<any>(ELEMENT_DATA);
+    public dataSource = new MatTableDataSource<TableSourceInterface>(
```

Comment:
I think it's a wrong type for data source. The Material table expects type of a row, and it stores and operates on arrays of those types. So it should be a type that represents a single row, a union betweeen single schema field and single data object.

Reply:
Changed on  DataSchemaField | ObjectInterface